.buildkite/steps/build-docker-image.sh

```diff
@@ -54,7 +54,7 @@ codename="${3:-}"
 version="${4:-}"
 push="${PUSH_IMAGE:-true}"
 
-if [[ ! "$variant" =~ ^(alpine|ubuntu|centos|sidecar)$ ]] ; then
+if [[ ! "$variant" =~ ^(alpine|ubuntu-18\.04|ubuntu-20\.04|centos|sidecar)$ ]] ; then
```

Comment:
I debated calling the variants and folders "ubuntu-bionic" and "ubuntu-focal" to have fewer escaping concerns where the names are used, but decided against it as the variant name is used on in the publishing script which I wanted to avoid changing too much.